examples/stm32/xively_demo_stm32f4xx_nucleo_eth/Src/xively_client.c

```diff
@@ -0,0 +1,906 @@
+/******************************************************************************
+ *                                                                            *
+ *  xively_client.c                                                           *
+ *                                                                            *
+ ******************************************************************************/
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <time.h>
+#include "cmsis_os.h"
+//#include "ms_httpserver.h"
+#include "xively.h"
+#include "xively_client.h"
+//#include "test_alloc.h"
+#include "xi_bsp_rng.h"
+#include "xi_bsp_time.h"
+
+/******************************************************************************
+ *                                                                            *
+ *  Macros                                                                    *
+ *                                                                            *
+ ******************************************************************************/
+/*
+ *  Application Specific
+ */
+#define XC_STR_LEN 64
+#define XC_CONNECT_TO 0    /* Seconds. Client/Broker connection wait time.   */
+#define XC_KEEPALIVE_TO 10 /* Seconds. Client/Broker keep-alive period.      */
+#define XC_CHECK_PERIOD 1  /* Seconds. Check period for publication changes. */
+#define XC_MAX_PUBS_IN_FLIGHT 1
+
+#define XC_TASK_STACK ( 512 * 6 )
+#define XC_TASK_NAME ( ( signed char* )"XC Thread" )
+
+/*
+ *  Device Specific
+ *  Credential Information
+ */
+#define MS_DEVICE 0
+#define BB_DEVICE 1
+
+#define BUILD_DEVICE BB_DEVICE
+
+#define XC_ACCOUNT_ID "bc4ebe58-b070-437b-9a64-a112bb2283d8"

```

Comment:
I don't think that we should put these credentials into the public repository... but haven't we putted them already ?